examples/shared-modules/README.md

```diff
@@ -0,0 +1,13 @@
+# Example app using shared modules
+
+This example features:
+
+* An app with two pages which has a common Counter component
+* That Counter component maintain the counter inside it's module.
```

Comment:
Maybe say something along the lines: the common plugin maintains state that is initialized only once.